miro_preprocessor/miro_copy_s3_asset/miro_copy_s3_derivative_asset/src/requirements.txt

```diff
@@ -0,0 +1 @@
+wellcome_lambda_utils==0.0.0.4
```

Comment:
what is this version number

Reply:
This was a bogus version number while I was trying to figure out how to publish a package in python